### PR TITLE
feat[g3.1-g3.5]: Basic Appointment Search and Appointment Search with Priorities connected frontend with backend; Advanced search for available appointment

### PR DESCRIPTION
**Pull request by VasilijeBursac**
- This branch contains completed Appointment Search (Basic and With Priorities) - Frontend connected with Backend.

### CODE REVIEW

**Comment by VasilijeBursac** [comment]
[(5) g3.1: Search for available appointment [3]](https://trello.com/c/mY5GwcfY/41-5-g31-search-for-available-appointment-3)

**Comment by VasilijeBursac** [comment]
[(3) g3.2: Alternative appointments (priority - doctor) [2]](https://trello.com/c/nQ7uB9UY/42-3-g32-alternative-appointments-priority-doctor-2)

**Comment by VasilijeBursac** [comment]
[(3) g3.3: Alternative appointments (priority - time) [2]](https://trello.com/c/Af6vGIpt/43-3-g33-alternative-appointments-priority-time-2)

**Comment by VasilijeBursac** [comment]
[(2) g3.4: Search for available specialist appointment [2]](https://trello.com/c/cJQkTT9B/44-2-g34-search-for-available-specialist-appointment-2)

**Comment by VasilijeBursac** [comment]
[(2) g3.5: Advanced search for available appointment [1]](https://trello.com/c/aH2hW9ud/45-2-g35-advanced-search-for-available-appointment-1)

File: HealthcareSystemFrontends/GraphicalEditor/MainWindow.xaml.cs [hunk 1]
```diff
@@ -686,37 +686,86 @@ private void ShowMedicineSearchResultObjectOnMapButton_Click(object sender, Rout
...
+                MapObject selectedAppointmentRoomMapObject = _mapObjectController.GetMapObjectById(selectedAppointmentRoomId);
+                ShowSelectedSearchResultObjectOnMap(selectedAppointmentRoomMapObject);
+            }
         }
 
         private void ScheduleAppointmentButton_Click(object sender, RoutedEventArgs e)
         {
-
+           
         }
```
Comment (anciz) [inline HealthcareSystemFrontends/GraphicalEditor/MainWindow.xaml.cs:767]: Why is this left empty?
Will it be implemented in near future?

Comment (VasilijeBursac) [reply HealthcareSystemFrontends/GraphicalEditor/MainWindow.xaml.cs:767]: Yes, it is going to be implemented in the next PR. I will just paste code here from the old local repo.

**Review by anciz — APPROVED** [review]
Seems to be good.

**Review by ogikuzma — APPROVED** [review]

**Pull request merged**